test/test_torch.py

```diff
@@ -1718,37 +1648,9 @@ def maybe_squeeze_result(l, r, result):
     def test_broadcast_batched_matmul(self):
         self._test_broadcast_batched_matmul(self, lambda t: t)
 
-    def test_matmul_out(self):
```

Comment:
why'd we get rid of test_matmul_out?

Reply:
No `matmul` with out argument

Reply:
Ok, added to the list of breaking changes.